### DISCUSSION TITLE
The name "Simple Icons"

### DISCUSSION THREAD

**Discussion by ineshbose** [simple-icons/simple-icons#7576] [General]
Hello,

I was just wondering what thought went into choosing the name Simple Icons for the project and if the name is very distinguishable or recognisable. For example, when I mention "Simple Icons" to someone, they wouldn't understand from the name that is a pack of brand icons, or when I search for "Font Awesome icons", I get the desired icons, but searching for "Simple icons" may not take me to the right place since it could mean [example 1](https://thenounproject.com/browse/icons/term/simple/) or [example 2](https://www.flaticon.com/free-icons/simple) (but searching for "svg icons for brands" at least leads one to this project).

Sorry for this question and I am in no way suggesting a rename/rebrand (which I understand would be really heavy).

The maintainers of this project have been very friendly and welcoming! 🙂

**Comment by sachinraja** [comment]
Yeah a rebrand is really off the table for us right now. I'm seeing our website and repository at the top of searches using Google though.
